juju/juju
# cmd/sla: Added new sla command

## Description of change

This pr adds a new client command:
`juju sla <level>`

This command is used to set the sla level for the model.
## QA steps
```bash
$ juju sla # run the juju sla command to see the default support level:
unsupported
$ juju sla essential  # set the sla level to essential
$ juju sla
essential
```
## Documentation changes

Ask @cmars.

## Bug reference

None
